--- alien/Alien/Package/Rpm.pm.orig	2005-07-19 22:14:31.000000000 +0200
+++ alien/Alien/Package/Rpm.pm	2005-10-17 12:14:32.000000000 +0200
@@ -292,7 +292,7 @@
 	print OUT "Requires: ".$this->depends."\n"
 		if defined $this->depends && length $this->depends;
 	print OUT "Summary: ".$this->summary."\n";
-	print OUT "Copyright: ".$this->copyright."\n";
+	print OUT "License: ".$this->copyright."\n";
 	print OUT "Distribution: ".$this->distribution."\n";
 	print OUT "Group: Converted/".$this->group."\n";
 	print OUT "\n";
